docs/core/entity/media-player.md

```diff
@@ -118,3 +118,25 @@ Optional. What type of media device is this. It will possibly map to google devi
 | tv | Device is a television type device.
 | speaker | Device is speakers or stereo type device.
 | receiver | Device is audio video receiver type device taking audio and outputting to speakers and video to some display.
+
+### Proxy album art for media browser
+
+Optional. Some media players can serve album artwork or other images on an internal network accessible to Home Assistant that is inaccessible from an external url.
+
+To do so, implement the `async_get_browse_image` method. It takes the `media_content_type`, `media_content_id`, and `browse_image` `get` parameters. You may want to also create a method that generates the appropriate url and use it as part of your `async_browse_media` implementation. Note: `/api/media_player_proxy/{entity_id}` requires a `token` get parameter. The value of this token can be retrieved using the `access_token` property implemented by the `MediaPlayerEntity` base class.
```

Comment:
```suggestion
To proxy an image via Home Assistant, set the `thumbnail` property of a `BrowseMedia` item to a url generated by the `self.get_browse_image_url(media_content_type, media_content_id, media_image_id=None)` method. The browser will then fetch this url, which will result in a call to `async_get_browse_image(media_content_type, media_content_id, media_image_id=None)`.

In `async_get_browse_image`, use `self._async_fetch_image(url)` to fetch the image from the local network.

> Do not pass a url as `media_image_id`. This can allow an attacker to fetch any data from the local network.
```